stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -70,33 +64,35 @@
     static final String GET = "GET";
     static final String POST = "POST";
 
+    private static final String LIVE_API_BASE = "https://api.stripe.com";
+    private static final String LIVE_LOGGING_BASE = "https://q.stripe.com";
+    private static final String LOGGING_ENDPOINT = "https://m.stripe.com/4";
+
+    private static final String CHARSET = "UTF-8";
+    private static final String TOKENS = "tokens";
+    private static final String SOURCES = "sources";
     private static final String DNS_CACHE_TTL_PROPERTY_NAME = "networkaddress.cache.ttl";
     private static final SSLSocketFactory SSL_SOCKET_FACTORY = new StripeSSLSocketFactory();
 
-    /**
-     * Create a {@link Source} using the input {@link SourceParams}.
-     *
-     * @deprecated As of v4.0.3 - will not be carried into 5+ and beyond
-     *
-     * @param sourceParams a {@link SourceParams} object with {@link Source} creation params
-     * @param publishableKey an API key
-     * @return a {@link Source} if one could be created from the input params,
-     * or {@code null} if not
-     * @throws AuthenticationException if there is a problem authenticating to the Stripe API
-     * @throws InvalidRequestException if one or more of the parameters is incorrect
-     * @throws APIConnectionException if there is a problem connecting to the Stripe API
-     * @throws APIException for unknown Stripe API errors. These should be rare.
-     */
-    @Deprecated
-    @Nullable
-    public static Source createSourceOnServer(
-            @NonNull SourceParams sourceParams,
-            @NonNull String publishableKey)
-            throws AuthenticationException,
-            InvalidRequestException,
-            APIConnectionException,
-            APIException {
-        return createSourceOnServer(sourceParams, publishableKey, null);
+    public static void logApiCall(
+            @NonNull Map<String, Object> loggingMap,
+            @Nullable RequestOptions options,
+            @Nullable LoggingResponseListener listener) {
+        if (options == null) {
+            return;
+        }
+
+        if (listener != null && !listener.shouldLogTest()) {
+            return;
+        }
+
+        String apiKey = options.getPublishableApiKey();
+        if (apiKey == null || apiKey.trim().isEmpty()) {
+            // if there is no apiKey associated with the request, we don't need to react here.
+            return;
+        }
+
+        fireAndForgetApiCall(loggingMap, LIVE_LOGGING_BASE, GET, options, listener);
```

Comment:
Just double checking, this is indeed a `GET` with `https://q.stripe.com`? I'm just referencing the `POST` with `https://m.stripe.com/4`.